src/js/app.js

```diff
@@ -12,19 +12,20 @@ function Result(props) {
   }
   return <div id="answer">{message}</div>;
 }
+
 // this JavaScript snippet appenended to src/js/app.js
 function App() {
   // this JavaScript snippet is later referred to as <<react-state>>
   const [epsilon, setEpsilon] = React.useState(0.001);
   // this JavaScript snippet is appended to <<react-state>>
   function onEpsilonChange(event) {
-    setEpsilon(event.target.value);
+    setEpsilon(event.target.value*1);
```

Comment:
Can you use Number() instead of `*1` here?

Reply:
Merged master to correct